### PR TITLE
Feature/ev 239 update dev setup

### PR DESCRIPTION
**Pull request by turegjorup**
https://jira.itkdev.dk/browse/EV-239

* Update to latest itk docker setup
* Support symfony cli
* Migrate from CircleCI (no longer worked) to Github Actions
* Update git commit hooks to use docker (needs work, fails in image)
* Update and apply coding standards to ensure check passes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,6 +107,12 @@ API documentation
 
 Go to http://event-database-api.vm/api/doc to see automatically generated API documentation.
 
+The API spec is also exported as `/web/api/api-spec-v1.json`. A job in github actions
+checks if there are changes to the api spec and fails if there is.
+
+If your changes are intentional you can re-export an updated API spec by running
+`docker compose exec phpfpm composer update-api-spec` and commint the changes.
```
Comment (rimi-itk) [inline README.md]: Typo in “commint”.

File: app/DoctrineMigrations/Version20190215035039.php [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
 declare(strict_types=1);
 
+/*
+ * This file is part of Eventbase API.
+ *
+ * (c) 2017–2018 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
```
Comment (rimi-itk) [inline app/DoctrineMigrations/Version20190215035039.php:12]: We should generally, i.e. in all our projects, consider removing this header or at least remove the the end year from it – or update and maintain it.

File: phpunit.xml.dist [hunk 3]
```diff
@@ -10,6 +10,7 @@
     <php>
         <ini name="error_reporting" value="-1" />
         <server name="KERNEL_CLASS" value="AppKernel" />
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled" />
```
Comment (rimi-itk) [inline phpunit.xml.dist:13]: 🎉

File: README.md [hunk 4]
```diff
@@ -74,13 +43,13 @@ if (isset($_SERVER['HTTP_CLIENT_IP'])
 
 ```sh
 # Get url (with port) from docker
-export event_database_api_url="http://event-database-api.local.computer:$(docker-compose port reverse-proxy 80 | cut -d: -f2)"
+export event_database_api_url="http://event-database-api.local.computer:$(docker compose port reverse-proxy 80 | cut -d: -f2)"
```
Comment (rimi-itk) [inline README.md]: This domain should be changed to `event-database-api.local.itkdev.dk` or we should use something along the lines of

```sh
export event_database_api_url="http://$(docker compose port reverse-proxy 80)"
```

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: app/DoctrineMigrations/Version20190215035039.php (hunk 2 above)
Comment (turegjorup) [reply app/DoctrineMigrations/Version20190215035039.php:12]: Aggreed. It was added by running `composer apply-coding-standards` so I left for now to limit scope of PR.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**